src/client/app/components/HeaderComponent.jsx

```diff
@@ -38,8 +39,9 @@ export default function HeaderComponent(props) {
 				<h1 style={titleStyle}>Open Energy Dashboard</h1>
```

Comment:
The title should also be resized to fit smaller windows, this and the logo take up an exorbitant amount of space in small windows.